### PR TITLE
feat: Open-Source Strategy Constraints

### PR DESCRIPTION
**Pull request by ivarconr**
This commits lift the flags and makes Strategy Constraints available to all users of Unleash.

Read more in the docs:
https://docs.getunleash.io/advanced/strategy_constraints

Enjoy!

@ivarconr
Author and maintainer

### CODE REVIEW

**Review by gastonfournier — APPROVED** [review]
:rocket:

File: website/docs/advanced/strategy-constraints.md [hunk 1]
```diff
@@ -4,7 +4,8 @@ title: Strategy Constraints
 ---
 
 :::info Availability
-Strategy constraints are available to Unleash Pro and Enterprise users.
+Strategy constraints are available to Unleash Pro and Enterprise users. 
+It is scheduled to be made available to Open Source users from **Unleash v4.16**.
```
Comment (ivarconr) [inline website/docs/advanced/strategy-constraints.md:8]: @thomasheartman Your input please :D

Comment (thomasheartman) [reply website/docs/advanced/strategy-constraints.md:8]: Nice! I'd suggest just leaving this as it is for now, however: I've got a more comprehensive doc PR in #2113, which also updates notes for custom context fields (and other instances of constraints).

What do you think?

```suggestion
```

Comment (thomasheartman) [reply website/docs/advanced/strategy-constraints.md:8]: So ideally, either merge the PRs, or revert the change you made to avoid conflicts.

Comment (ivarconr) [reply website/docs/advanced/strategy-constraints.md:8]: I will revert my change!

**Review by thomasheartman — APPROVED** [review]
Looks great to me! Super happy to see this land 😄

**Review by chriswk — APPROVED** [review]
Nice :)

**Pull request merged**